sdk/communication/azure-communication-chat/README.md

```diff
@@ -160,18 +160,14 @@ String chatThreadId = "Id";
 ChatThreadClient chatThreadClient = chatClient.getChatThreadClient(chatThreadId);
 ```
 
-#### Update a chat thread
+#### Update a chat thread topic
 
-Use `updateChatThread` method to update a thread's properties
-`updateChatThreadOptions` is used to describe the property change of the chat thread.
+Use `updateTopic` method to update a thread's topic
+`topic` is used to hold the new topic of the thread.
 
-- Use `topic` to give chat thread a new topic;
-
-<!-- embedme ./src/samples/java/com/azure/communication/chat/ReadmeSamples.java#L131-L133 -->
+<!-- embedme ./src/samples/java/com/azure/communication/chat/ReadmeSamples.java#L131-L131 -->
 ```Java
-UpdateChatThreadOptions updateChatThreadOptions = new UpdateChatThreadOptions()
```

Comment:
If UpdateChateThreadOptions is no longer used, shouldn't it be removed from the model or at least not to expose it?

To make this consistent, should the swagger file be changed and the underlying layer regenerated?

Reply:
Also, I think the high level point is around the visibility of this API, if we want to keep it and also keep it public then there is no harm in having two separate methods for update (maybe the options are supposed to do more than the only  topic update in future).. but if that is not the intension then the right thing would be remove it from the model altogether and clean up the references in swagger readme as well.

Reply:
@juancamilor what are your thoughts here?

Reply:
yeah, I think we should remove it as well. The good point about future changes from @ankitarorabit  was brought up during the reviews with the Azure folks and the decision was to aim for a better user experience by having the specific updateTopic method. (also because at this moment, we don't foresee any need for the updates of other attributes in the thread).

Reply:
yeah, I think we should remove it as well. The good point about future changes from @ankitarorabit was brought up during the reviews with the Azure folks and the decision was to aim for a better user experience by having the specific updateTopic method. (also because at this moment, we don't foresee any need for the updates of other attributes in the thread).